Base/usr/share/man/man7/kernel_cmdline.md

```diff
@@ -0,0 +1,129 @@
+## Name
+
+Kernel cmdline - boot arguments to control system functionality
+
+## Description
+
+The kernel supports boot parameters to change some of its behavior. This
+functionality can be used to disable or enable features in the OS entirely, but
+can sometimes be used also to change a feature's behavior without disabling it
+entirely.
+
+Some of the boot parameters require a value after it. In that case, you will
+need to write them like this:
+
+```
+param1=value ...
+
+```
+For those that don't require, you should just write them without equals sign,
+like that:
+
+```
+param1 ...
+
+```
+
+## Where to put those parameters
+
+You can either add kernel boot parameters when you invoke `./run`:
+
+```
+./run qcmd param1 param2 ...
+
+```
+for Q35 machines, use this layout:
+
+```
+./run q35_cmd param1 param2 ...
+
+```
+
+You may wish to write the kernel boot parameters in a more consistent method.
+For that, you can modify the GRUB configuration files. Since we have 3 of them
+(one for each partition scheme), you should pick the right one for you, so if
+you build a virtual disk with a GPT partition scheme, you should edit
+`grub-gpt.cfg` file.
+
+## Kernel boot parameters list
+
+Since it's pretty much not convenient for beginners to have a travel in the
```

Comment:
"pretty much not convenient" -> "not very convenient" or just "inconvenient"

What is "to have a travel in the file" supposed to mean? Maybe "to browse the source"?

Reply:
Generally, this is not the kind of language you'd find in man pages. I'd simply say, "The following arguments are recognized:"

Reply:
I see the language should be more "technical". I'll take care of that.